src/main/webapp/index-script.js

```diff
@@ -36,9 +37,20 @@ function loadMainButtons() {
   });
 
   logInButton.addEventListener("click", () => {
-    window.location.href = "/login"
-  });
+    window.location.href="/login" 
+  }); 
+}
+
+function login(){
```

Comment:
Comment function purpose above function header. Not entirely sure whether this function redirects users to log-in page or if it just retrieves the login status. If the latter is true, a more descriptive function header could be helpful as well.

Reply:
Check last commit - this was already fixed!